apps/prairielearn/src/lib/code-caller/code-caller-native.ts

```diff
@@ -11,45 +11,47 @@ import { logger } from '@prairielearn/logger';
 import { deferredPromise } from '../deferred.js';
 import { APP_ROOT_PATH, REPOSITORY_ROOT_PATH } from '../paths.js';
 
-import { FunctionMissingError } from './code-caller-shared.js';
+import { type CodeCallerNativeChildProcess } from './code-caller-native-types.js';
+import {
+  FunctionMissingError,
+  type CodeCallerState,
+  CREATED,
+  WAITING,
+  IN_CALL,
+  RESTARTING,
+  EXITING,
+  EXITED,
+  type CodeCaller,
+  type PrepareForCourseOptions,
+  type CallType as CallTypeShared,
+} from './code-caller-shared.js';
+
+/* Re-export types until executor is in TypeScript are merged */
```

Comment:
Hopefully we can land the executor-in-TS PR first!

Reply:
That is in #11011, which just got merged in.